pkg/engine2/operational_eval/resource_template_mock_test.go

```diff
@@ -0,0 +1,299 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
This is caused by the go generate still pointing to `engine2` - can you remove these after you clean up the generate?

Reply:
ah yeah ill clean this up